test/unit/helpers/curly-invocation-component-test.js

```diff
@@ -18,6 +18,10 @@ describe('#transformTagName', function() {
     expect(transformTagName('foo@bar-baz')).to.equal('Foo@BarBaz');
     expect(transformTagName('foo/bar-baz')).to.equal('Foo::BarBaz');
     expect(transformTagName('foo/bar-baz/bang')).to.equal('Foo::BarBaz::Bang');
+    expect(transformTagName('foo$bar-bang')).to.equal('Foo$BarBang');
+    expect(transformTagName('foo$bar-bang::baz::foo3bar')).to.equal('Foo$BarBang::Baz::Foo3bar');
+    expect(transformTagName('foo$bar-bang::baz??foo3bar')).to.equal('Foo$BarBang::Baz??Foo3bar');
+    expect(transformTagName('foo$bar-bang::baz/foo3bar')).to.equal('Foo$BarBang::Baz::Foo3bar');
```

Comment:
was `Foo-Bar` a valid component name before? I'm wondering if we should add test cases for that too 🤔

Reply:
@Turbo87 - you mean in curlies?

Reply:
+1 on @rwjblue's question. If you mean in curlies, yes I think it is supported, I can add a test case for that.

Reply:
yeah, I would assume that `{{Foo-Bar}}` would be a valid invocation for `components/foo-bar.js`, right?

I'm asking because I noticed that `SIMPLE_DASHERIZE_REGEXP` only considered lowercase characters and was curious if that's correct or not :)

Reply:
@Turbo87 added the test case :)

Reply:
sweet, thanks!